layers/imagick/Dockerfile

```diff
@@ -35,9 +35,9 @@ RUN make install
 
 # Compile the php imagick extension
 WORKDIR ${IMAGICK_BUILD_DIR}
-RUN pecl download imagick
-RUN tar xzf imagick-3.4.4.tgz
-WORKDIR ${IMAGICK_BUILD_DIR}/imagick-3.4.4
+RUN git clone https://github.com/Imagick/imagick
+WORKDIR ${IMAGICK_BUILD_DIR}/imagick
+RUN git checkout 132a11fd26675db9eb9f0e9a3e2887c161875206
```

Comment:
Hm.. This will install an unreleased version of Imagick. Is that safe?

Reply:
I would be fine with this if we used a stable release on PHP7

Reply:
Oh, there is quite tough convo about that in issues: https://github.com/Imagick/imagick/issues/358 so.. I don't know, tests looks fine.

Sury is also compiling it from master. Also: https://github.com/oerdnj/deb.sury.org/issues/1502

Reply:
@Nyholm can I use `RUN if [[ "$PHP_VERSION" = "74" ]]` hack or there is neater way to do it?

Reply:
That is something I had in mind, yes.

Reply:
@Nyholm done